test/sessionAccessTokenSigningKeyUpdate.test.js

```diff
@@ -483,10 +470,8 @@ describe(`sessionAccessTokenSigningKeyUpdate: ${printPath(
         assert.strictEqual(requestMock.callCount, 1);
     });
 
-    it("test reducing access token signing key update interval time", async function () {
-        const appId = "testapp-" + Date.now();
-        const connectionURI = await startST({
-            appId,
+    it.skip("test reducing access token signing key update interval time", async function () {
```

Comment:
Please add a comment about why we are skipping this and/or a task to re-enable it.